lib/generators/rspec/scaffold/templates/routing_spec.rb

```diff
@@ -5,41 +5,73 @@
   describe "routing" do
 <% unless options[:singleton] -%>
     it "routes to #index" do
-      expect(:get => "/<%= ns_table_name %>").to route_to("<%= ns_table_name %>#index")
+      if RUBY_VERSION > '1.9.3'
```

Comment:
This check, and keeping the old syntax is unnecessary, edge [`rspec-rails` supports Rails 4.2+](https://github.com/rspec/rspec-rails/blob/master/rspec-rails.gemspec#L36), and they [only support Ruby 1.9.3+](https://github.com/rails/rails/blob/4-2-stable/rails.gemspec#L10).